irony-cdb.el

```diff
@@ -125,6 +125,11 @@ for files that it cannot handle."
                             (caar it)
                             (cdar it)))))))
 
+(defun irony-cdb--remove-compiler-from-flags (flags)
+  "Removes the compiler from flags read from a compilation database. In the
+future this might get more invloved."
```

Comment:
Seconde sentence should be separated by 2 newlines.

Also invloved -> involved.